system_test/aest_sync_SUITE.erl

```diff
@@ -428,13 +426,15 @@ net_split_recovery(Cfg) ->
     ok.
 
 quick_start_stop(Cfg) ->
-    setup_nodes(cluster([n1, n2], #{}), Cfg),
-    start_node(n2, Cfg),
-    start_node(n1, Cfg),
+    Nodes = [n1, n2],
+    setup_nodes(cluster(Nodes, #{}), Cfg),
+    [start_node(N, Cfg) || N <- Nodes],
+    Blocks = wait_for_value({height, 5}, Nodes, 5 * ?MINING_TIMEOUT, Cfg),
     stop_node(n2, 2000, Cfg),
```

Comment:
This will never fail because the node got time to start properly. What I tested was:
 - Start both nodes; stop node 2 right away (should crash); restart node 2; check they synchronize.
 - Start both nodes; let them build some chain; stop node 2; start node 2 and stop it right away (couldn't make it crash); restart node 2; check they synchronize.

Reply:
I don't follow. What actual changes do you request?

Reply:
In your test, as I understand it, you build a chain before stopping node2; this gives time for node 2 to start so it will never crash.

Reply:
I described the scenarios I think that should be test.